api3_deprecated/index.html

```diff
@@ -1,16 +1,21 @@
 ---
 layout: api3
```

Comment:
Can we just delete this file/folder? URL has changed so users can't find it anyway + if we need it we can look through source control